src/components/dictionaryConcepts/components/DescriptionRow.jsx

```diff
@@ -124,7 +126,9 @@ class DescriptionRow extends Component {
             className=" btn btn-danger concept-form-table-link"
             id="remove-description"
             type="button"
-            onClick={event => this.handleRemove(event, this.props.newRow)}
+            onClick={event => ((this.props.newRowUuid === '')
```

Comment:
Why not instead check for null?

Reply:
Because the value am checking against is a string and null is of object type

Reply:
What is the difference between newRow and newRowUuid?

Reply:
newRow is used on the Edit concept interface to access a specific description row details and newRowUuid is used when on the create concepts page when accessing a specific description row id that is stored in a list called description.

Reply:
The Implementaion for the adding of a description row on the edit concept page and the create concept page are different. This is where the two different terms arise from.
When adding a new decription row on the edit concept page.
- This stores the row data in state as an object with key as Uuid so when removing the row we access the id as newRow.uuid

while
When adding a new decription row on the create concept page.
- This stores the row data in state as a list of UUIDs in form of strings so when removing the row we access the id directly

Hence the two different term